search_service/proxy/elasticsearch.py

```diff
@@ -23,6 +23,15 @@
 
 LOGGING = logging.getLogger(__name__)
 
+# mapping to translate request
+TABLE_MAPPING = {
+    'tag': 'tags',
+    'schema': 'schema_name.raw',
+    'table': 'name.raw',
+    'column': 'column_names.raw',
+    'database': 'database.raw'
```

Comment:
Are there any other implications for `.normalized`? On an initial attempt I couldn't find any clear doc on `.normalized` vs `.raw`.

Reply:
NB: `.normalized` isn't a field you get for free; we added a normalizer to the index_map.py file and added an additional field beyond `.raw`

Reply:
It would be really nice if we support case sensitive search for at least the tags. Because tags are the things people usually use the case sensitive acronyms i.e., like `PII` . Thoughts ?

Reply:
I agree with @verdan but I'm also happy with that change being a follow up (would also be a straight forward issue for a new contributor to work on)

Reply:
@verdan thanks for raising that. I filed an issue, let's continue the discussion there for all those interested. If we can decide on the exact requirements it will be easy for someone to pick up the task. https://github.com/lyft/amundsen/issues/283